Client/WarFare/UIImageTooltipDlg.cpp

```diff
@@ -1052,6 +1053,79 @@ int	CUIImageTooltipDlg::CalcTooltipStringNumAndWrite(__IconItemSkill* spItem, bo
 	}
 	ERROR_EXCEPTION
 
+	if (spItem->pItemBasic != nullptr)
+	{
+		std::string szRemark = spItem->pItemBasic->szRemark;
+	
+		int strSize = spItem->pItemBasic->szRemark.size();
+		if (strSize > 0)
+		{
+			int splitPos = 0;
+			for (int i = 0; i < strSize; i++)
+			{
+				if (szRemark[i] == '(' || (szRemark[i] == ' ' && i > strSize / 2))
```

Comment:
Officially this only splits on words dictated by spaces, so it ignores `(`, but so long as it's noted in a comment that it's not actually official behaviour, that's fine.

What it also does, however, is it enforces a minimum word limit of 4 for it to enforce the split, so it isn't splitting on short descriptions.